docs/reference/individual_learner_dashboard.rst

```diff
@@ -1,16 +1,76 @@
 Individual Learner Dashboard Reports
 ####################################
 
-The Individual Learner Dashboard Reports focus on relevant information about each learner’s performance and information, including course enrollment and activity logs, to give a general overview. If the name, username, and email address columns are empty in the learner summary, it is because your site operator has chosen to omit limited personally identifiable information from being displayed on your Open edX instance.
+The Individual Learner Dashboard Reports focus on relevant information about each learner’s performance and information, including course enrollment and activity logs, to give a general overview. Suppose the name, username, and email address columns are empty in the learner summary. In that case, your site operator has omitted limited personally identifiable information from being displayed on your Open edX instance. The Individual Learner Dashboard can only be filtered by one or more individual learners if the Operator has chosen to display limited personally identifiable information on your instance. 
 
+One particular benefit of this dashboard is that it gives the course delivery team a general view of the course performance while zooming in and out on each learner, combining the `Course Dashboard <https://docs.openedx.org/projects/openedx-aspects/en/latest/reference/course_overview_dashboard.html>`_ and `At Risk Learner Dashboard <https://docs.openedx.org/projects/openedx-aspects/en/latest/reference/learner_groups_dashboard.html>`_ information. As with the rest of the Aspects dashboards, this dashboard’s charts apply cross-filters to charts whose data sets contain the same name.
 
-.. image:: /_static/individual_dashboard.png
+.. image:: /_static/individual_learner_dashboard.png
 
-As with the rest of the Aspects dashboards, this dashboard's charts apply cross-filters to charts whose data sets contain the same name. 
+This dashboard's data information is divided into two sections: the Learner Summary and the pages, problems, videos, and help reports.
 
-One particular benefit of this dashboard is that it gives the course delivery team a general view of the course performance while zooming in and out on each learner, combining the Course Dashboard and At-Risk Learner Dashboard information.
+1. Learner Summary:
```

Comment:
I don't think we need the colons at the end of these, it looks a little weird since they get turned into headers